db/migrate/20150427183903_add_fields_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddFieldsToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :beard_length , :integer
+    add_column :users, :massaged , :boolean , :default => false
+    add_column :users, :last_massage_type , :string
```

Comment:
Looks like this is the reason for string result "ok"/"failed". It still seems wrong to me. We can easily show message "Ok" if result is true and "Failed" if result is false.